### DISCUSSION TITLE
How to show up bitmaps when revisiting editing OCR?

### DISCUSSION THREAD

**Discussion by mpek** [SubtitleEdit/subtitleedit#6340] [General]
I started to OCR from an mkv made from DVD with Bitmap/PGS german subtitle.  
When making a break and restarting computer and continuing to work on the project, by clicking on Spell check, how can I show up again the bitmaps as a reference when editing the text?
When starting this process as new I can see the bitmaps while editing the position in question.

**Comment by niksedk** [comment]
Try the context menu:

![image](https://user-images.githubusercontent.com/1261331/196471584-bd2bdf2f-8fd8-4e22-8ada-879288f35daa.png)

**Comment by mpek** [comment]
Ah okay, nice to know. Why is that hidden? :) 
Would need another hint: Where is that file being saved, when I imported the image based subtitle from a video file (mkv)?

**Comment by niksedk** [comment]
He, in case of mkv, you would need to right-click in the OCR list view and export subs...
